src/js/events/CosmosPackagesActions.js

```diff
@@ -18,14 +18,21 @@ import {
   REQUEST_COSMOS_REPOSITORY_DELETE_SUCCESS,
   REQUEST_COSMOS_REPOSITORY_DELETE_ERROR,
   REQUEST_COSMOS_PACKAGE_LIST_VERSIONS_SUCCESS,
-  REQUEST_COSMOS_PACKAGE_LIST_VERSIONS_ERROR
+  REQUEST_COSMOS_PACKAGE_LIST_VERSIONS_ERROR,
+  REQUEST_COSMOS_SERVICE_DESCRIBE_SUCCESS,
+  REQUEST_COSMOS_SERVICE_DESCRIBE_ERROR
 } from "../constants/ActionTypes";
 import AppDispatcher from "./AppDispatcher";
 import Config from "../config/Config";
 import Util from "../utils/Util";
 
-function getContentType(action, actionType, version = "v2") {
-  return `application/vnd.dcos.package.${action}-${actionType}+json;charset=utf-8;version=${version}`;
+function getContentType(
```

Comment:
Not blocking, but I'd like us to change the function interface so that we can pass an options object as it's starts to become a little messy.

Reply:
I agree, will change this to named object param like: `getContentType({ action, actionType, entity, version = "v2"})`. It makes sense to be explicit about `entity` and have no default because it is semantically similar to action and actionType.

Reply:
I wish there is refactoring tools for a change like this...doesn't seem to be a way besides manually changing 😢

Reply:
There are tools... one of the reasons I still use IntelliJ 😊

Reply:
I'll have to try out WebStorm, I didn't see any way to do this in VSCode. 👍